docs/conf.py

```diff
@@ -65,7 +65,6 @@
     "seaborn",
     "skimage",
     "svgutils",
-    "templateflow",
```

Comment:
Hopefully we don't end up downloading things during builds, but if so, so be it.